code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -197,6 +198,62 @@
 	. = ..()
 	fill_cigarre_package(src, list("fuel" = 15))
 
+/obj/item/weapon/storage/fancy/cigarettes/homeless
+	name = "\improper Nomads packet"
+	desc = "A packet of six Nomads cigarettes. Nomads's Extra strong for when your life is more extra hard"
```

Comment:
Reenglish?

Reply:
WIP until someone brings up a good desc, btw it was from SuicidalSquirrel.